crates/aranya-policy-lang/src/lang/parse/policy.pest

```diff
@@ -42,9 +42,11 @@ vtype = _{ string_t | bytes_t | int_t | bool_t | id_t | struct_t | enum_t | opti
 // A field definition is an identifier followed by a type. Used in
 // struct definitions, function arguments, etc.
 field_definition = { identifier ~ vtype }
+// struct field insertion. Copy fields from another struct
+struct_field_insertion = { "+" ~ identifier }
 // A field definition list is a list of field definitions. Trailing
 // commas are allowed in any list.
-field_definition_list = _{ field_definition ~ ("," ~ field_definition)* ~ ","? }
+field_definition_list = _{ (field_definition | struct_field_insertion) ~ ("," ~ (field_definition | struct_field_insertion))* ~ ","? }
```

Comment:
Just realized this is not the right place to add field insertion, because this rule is used in a number for places that should not allow that, e.g. fact signatures, and function args.

Reply:
Pushed commit to resolve this.